src/vsg/io/VSG.cpp

```diff
@@ -275,7 +275,7 @@ bool VSG::write(const vsg::Object* object, std::ostream& fout, ref_ptr<const Opt
 
 bool VSG::getFeatures(Features& features) const
 {
-    features.extensionFeatureMap["vsgb"] = static_cast<FeatureMask>(READ_FILENAME | READ_ISTREAM | READ_MEMORY | WRITE_FILENAME | WRITE_OSTREAM);
-    features.extensionFeatureMap["vsgt"] = static_cast<FeatureMask>(READ_FILENAME | READ_ISTREAM | READ_MEMORY | WRITE_FILENAME | WRITE_OSTREAM);
+    features.extensionFeatureMap["vsgb"] = static_cast<FeatureMask>(static_cast<uint8_t>(FeatureMask::READ_FILENAME) | static_cast<uint8_t>(FeatureMask::READ_ISTREAM) | static_cast<uint8_t>(FeatureMask::READ_MEMORY) | static_cast<uint8_t>(FeatureMask::WRITE_FILENAME) | static_cast<uint8_t>(FeatureMask::WRITE_OSTREAM));
```

Comment:
This screams out a regression in readability and usability.  Really WTF.  If you ever wanted an illustration of why some C++ features should be used with care this is it.  Code should be made more expression and readable,